spec/index.html

```diff
@@ -259,28 +255,100 @@ <h2>A Canonical form of N-Quads</h2>
+      <li>The token <code><a href="#grammar-production-EOL">EOL</a></code> MUST be a single <code>U+000A</code>.</li>
+      <li>The final <code><a href="#grammar-production-EOL">EOL</a></code> MUST be provided.</li>
   </ul>
-  <div class="issue" data-number="2">
-    Note open errata:
-    <ul>
-      <li><a href="https://www.w3.org/2001/sw/wiki/RDF1.1_Errata#erratum_32">32 – Issues with N-Quads/N-Triples canonicalization</a>.</li>
-      <li><a href="https://www.w3.org/2001/sw/wiki/RDF1.1_Errata#erratum_33">33 – Ambiguity of canonical N-Triples</a></li>
-    </ul>
+
+  <div class="issue" data-number="16">
+    <p>Re-consider the use of `UCHAR` and `ECHAR` escapes in N-Triples/N-Quads canonicalization.
+      The 1.1-based recommendation prohibits the use of `UCHAR` (`U+XXXX`)
+      and allows `ECHAR` only for `U+0022` (quote `\"`),
+      `U+005C` (backslash `\\`),
+      `U+000A` (<code title="LINE FEED"><sub>LF</sub></code> `\n`),
+      and `U+000D` (<code title="CARRIAGE RETURN"><sub>CR</sub></code> `\r`).
+      However, the use of control characters can obfuscate text when presented,
+      creating a potential security concern.</p>
+
+    <p>A future version may consider requiring all characters between
+      `U+0000` and `U+001F` (other than `U+000A` (<code title="LINE FEED"><sub>LF</sub></code>)
+      and `U+000D` (<code title="CARRIAGE RETURN"><sub>CR</sub></code>))
+      to be represented using `UCHAR`.</p>
   </div>
 </section>
 
+<section id="privacy-considerations">
+  <h2>Privacy Considerations</h2>
+  <p class="ednote">TODO</p>  
+</section>
+
+<section id="security">
+  <h2>Security Considerations</h2>
+
+  <p>The <a href="#grammar-production-STRING_LITERAL_QUOTE">STRING_LITERAL_QUOTE</a>
+    production allows the use of unescaped control characters.
+    Although this specification does not directly expose this content to an end user,
+    it might be presented through 
+    If presented through a user agent, this may cause the </p>
+
+  <p>N-Quads is a general-purpose assertion language;
+    applications may evaluate given data to infer more assertions or to dereference IRIs,
+    invoking the security considerations of the scheme for that IRI.
+    Note in particular, the privacy issues in [[!RFC3023]] section 10 for HTTP IRIs.
+    Data obtained from an inaccurate or malicious data source may lead to inaccurate or misleading conclusions,
+    as well as the dereferencing of unintended IRIs.
+    Care must be taken to align the trust in consulted resources with the sensitivity of
+    the intended use of the data;
+    inferences of potential medical treatments would likely require different trust than inferences
+    for trip planning.</p>
+
+  <p>N-Quads is used to express arbitrary application data; security considerations
+    will vary by domain of use. Security tools and protocols applicable to text
+    (for example, PGP encryption, MD5 sum validation, password-protected compression)
```

Comment:
```suggestion
    (for example, PGP encryption, checksum validation, password-protected compression)
```
Don't name a algorithm, especially one that is no longer recommended for secure use.